src/composables/trade/useSor.ts

```diff
@@ -202,8 +201,6 @@ export default function useSor({
   }
 
   async function handleAmountChange(): Promise<void> {
-    resetState();
```

Comment:
Is this definitely not needed anymore? It was hiding a price impact alert if it was shown.

Reply:
Yeah I just moved it to sit inside `useTrading` (sometimes I might want to call `handleAmountChange` without resetting the state - so this is why I lifted it up)